sdk/storage/azure-storage-blob/azure/storage/blob/_generated/operations/_blob_operations.py

```diff
@@ -3628,6 +3632,9 @@ def copy_from_url(  # pylint: disable=inconsistent-return-statements
         :param copy_source_authorization: Only Bearer type is supported. Credentials should be a valid
          OAuth access token to copy source. Default value is None.
         :type copy_source_authorization: str
+        :param copy_source_tags: Optional, default 'replace'.  Indicates if source tags should be
```

Comment:
nit: Remove one of the whitespaces between the period and the capital I in 'Indicates'

Reply:
This is generated code so unfortunately, I cannot change it :)